src/views/open-api-docs-preview-v2/utils/get-props-from-preview-data.ts

```diff
@@ -38,7 +41,23 @@ export default async function getPropsFromPreviewData(
 	// and prefer to have content updates made at the content source... but
 	// some shims are used often enough that they feel worth including in the
 	// preview too. Namely, shortening to `HCP` in the spec title.
-	const schemaTransforms = [schemaTransformShortenHcp]
+	const schemaTransforms = [
```

Comment:
I've added "shorten protobufAny" transforms to the preview tool, to make it easier to compare content. These could be removed later if they don't feel right.